README.md

```diff
@@ -262,6 +262,15 @@ required plugins.
   * Use the "core-modules" profile if working on the Pulsar core modules since the source code for those modules can be compiled in IntelliJ.
   * Sometimes it might help to mark a specific project ignored in IntelliJ Maven UI by right-clicking the project name and select **Ignore Projects** from the menu.
   * Currently, it is not always possible to run unit tests directly from the IDE because of the compilation issues. As a workaround, individual test classes can be run by using the `mvn test -Dtest=TestClassName` command.
+    
+* The above steps have all been performed, but a test still won't run.
+  * In this case, try the following steps:
+    * Close IntelliJ
+    * Run `mvn clean install -DskipTests` on the command line
+    * Reopen IntelliJ
```

Comment:
```suggestion
    1. Close IntelliJ.
    2. Run `mvn clean install -DskipTests` on the command line.
    3. Reopen IntelliJ.
```

Use an ordered (numbered) list when the sequence is significant, for example, when writing procedures or ranking items. If the items in a list represent rules or other types of information that you want to refer to, you can refer to them by number. 

https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.yoetnldi5amd